README.md

```diff
@@ -209,6 +210,73 @@ our trained model. For example:
 
 ![history](http://i.imgur.com/y45tum4.png)
 
+### Estimating Customers' Lifetime Value
+
+For this whole time we didn't take into account the economic value of each transaction and we focused mainly on
+transactions' occurences. To estimate this we can use the Gamma/Gamma model. But first we need to create summary data
+from transactional data also containing economic values for each transaction (i.e. profits or revenues).
+
+    from lifetimes.datasets import load_transaction_data_with_monetary_value
+
+    summary_with_money_value = load_transaction_data_with_monetary_value()
+    summary_with_money_value.head()
+    returning_customers_summary = summary_with_money_value[summary_with_money_value['frequency']>0]
```

Comment:
Literature (Fader et al. 2005) uses x > 0, but the main reason I do it now is that while computing p_0 I need to compute `np.log(special.gamma(x))` which would be +inf if freq was to be 0. Otherwise I could be using laplacian smoothing (e.g. adding a +1 to all the frequencies, but I am not sure if this is a sound mathematical choice, what do you think?).